### DISCUSSION TITLE
Can't reach database server at aws-0-eu-central-1.pooler.supabase.com:6543

### DISCUSSION THREAD

**Discussion by henriquestiagoo** [prisma/prisma#25320] [Q&A]
### Question

Hi,

I am getting the error below when I am connected to a specific wi-fi network (university network). When I am using other networks, I run with no problems.

```
digest: "3580489738"
 ⨯ PrismaClientInitializationError: 
Invalid `prisma.restaurant.findMany()` invocation:


Can't reach database server at `aws-0-eu-central-1.pooler.supabase.com:6543`

Please make sure your database server is running at `aws-0-eu-central-1.pooler.supabase.com:6543`.
```

These are my connection strings I am are using for DATABASE_URL and DIRECT_URL in my .env file.

```
DATABASE_URL="postgresql://postgres.neppzfiudskbcojbjxxx:[PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres?pgbouncer=true&connection_limit=1" 
DIRECT_URL="postgresql://postgres.neppzfiudskbcojbjxxx:[PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:5432/postgres"  
```

What can be the problem here ? Any help ?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url = env("DATABASE_URL")
  directUrl = env("DIRECT_URL")
}
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
macOS

- Database: 
postgres

- Node.js version: 
v20.9.0

- Run `prisma -v` to see your Prisma version and paste it
prisma                  : 5.20.0
@prisma/client          : 5.20.0
Computed binaryTarget   : darwin-arm64
Operating System        : darwin
Architecture            : arm64
Node.js                 : v20.9.0
Query Engine (Node-API) : libquery-engine 06fc58a368dc7be9fbbbe894adf8d445d208c284 (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 06fc58a368dc7be9fbbbe894adf8d445d208c284 (at node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.20.0-12.06fc58a368dc7be9fbbbe894adf8d445d208c284
Default Engines Hash    : 06fc58a368dc7be9fbbbe894adf8d445d208c284
Studio                  : 0.502.0

**Comment by ludralph** [comment]
Hi @henriquestiagoo 👋 

The university network might have strict firewall rules and restrictions that might block certain ports or types of traffic. You can try using a VPN to bypass the University Network restriction.

**Reply by KuhlTime** [reply]
I have tried using a VPN from all sort of locations, but the error persists.

**Reply by ludralph** [reply]
Hi @KuhlTime 

You might want to try adding a connection timeout to your DATABASE_URL. As suggested in another [GitHub comment](https://github.com/prisma/prisma/issues/13807#issuecomment-1457867408), you can add `?connect_timeout=300` at the end of your connection string. Ensure that the region in your connection string (aws-0-eu-central-1) matches the actual region of your Supabase project. As mentioned in a [GitHub comment](https://github.com/prisma/prisma/issues/21740#issuecomment-1457867408), sometimes the region in the provided connection string might be incorrect.  If possible, try using a mobile hotspot or another network to confirm if the issue is specific to the university network.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
